services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceParentHelixAdmin.java

```diff
@@ -624,6 +632,52 @@ private void sendAdminMessageAndWaitForConsumed(String clusterName, String store
     waitingMessageToBeConsumed(clusterName, storeName, message.executionId);
   }
 
+  @Override
+  public void deleteValueSchemas(String clusterName, String storeName, Set<Integer> unusedValueSchemaIds) {
+    Set<Integer> inuseValueSchemaIds = getInUseValueSchemaIds(clusterName, storeName);
```

Comment:
I assume this code path is for the admin operation, right?
Shall we get rid of the admin tool command to make it simpler?

Reply:
Sure, but first lets wait for @FelixGV comment on this.